Foundation/NSNotification.swift

```diff
@@ -187,8 +187,13 @@ open class NotificationCenter: NSObject {
             self._observers = _observers.filterOutObserver(observer, name: aName, object: object)
         })
     }
-    
+
+    @available(swift, obsoleted: 4.0)
```

Comment:
We can get client-side fixes if we annotate it with a 'renamed' attribute

`@available(*,unavailable,renamed:"addObserver(forName:object:queue:using:)")`